meta-intel-edison-bsp/recipes-kernel/bcm43340/bcm43340-addr_0.1.bb

```diff
@@ -0,0 +1,29 @@
+DESCRIPTION = "Daemon listening to Edison PWR long button press, and starting OOBE service when it happens"
```

Comment:
That's probably not the description you wanted to have for this one. Copy artifact?

Reply:
Yes, thanks for finding.